CMakeLists.txt

```diff
@@ -270,6 +271,12 @@ if(WIN32 AND NOT MINGW)
     )
 endif()
 
+if (USE_COLORS)
+    # If user wants terminal colors. This is mainly for log, but can be used elsewhere too.
+    add_definitions(-DUSE_COLORS)
+    message(STATUS "Enabled terminal colors")
+endif ()
```

Comment:
The if and endif should be near attached to the parens for consistency with other stuff in this file